Bika_Project X Polkadot

JhonnyAllen79 opened the pull request:
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying? 
- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [ ] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [ ] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [ ] The software delivered for this grant will be released under an open-source license specified in the application.
- [ ] The initial PR contains only one commit (squash and force-push if needed).
- [ ] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

JhonnyAllen79 commented:
Hi W3F, I'm Jhonny Allen from Bika Project. 
I have filled out the W3F grant registration form, I have filled in the project details, I hope this can help the verification and approval process for the Bika Project grant application, I hope W3F provides a grant for Bika development, if anyone wants to give feedback please reply to this message so I can improve.

Noc2 reviewed (changes requested):
Thanks for the new application. (for reference https://github.com/w3f/Grants-Program/pull/792). However, your PR currently contains two applications (application-BikaProject.md and applications/application-BikaProject.md). Could you please remove one of them? Also feel free to remove all the text from the template that isn’t in bold letters.

JhonnyAllen79 commented:
Hi @Noc2, thanks for the feedback regarding the BikaProject.md app, I've fixed the issue you mentioned, I'd like the BikaProject.md app to be reviewed soon. If any input please reply.

Regards
Bika Token/Bika StarX

Noc2 reviewed (changes requested):
Thanks for the update. However, your application still contains a lot of the text of the template, which makes it hard to read. Feel free to remove this or ideally even answer all the questions. Could you also move the application to the application folder? 

Regarding the deliverables itself, it looks like you want to create a DEX that is able to exchange Polkadot and BSC tokens. How exactly are you going to archive this? Don’t you need a bridge for this? The second milestone seems to implement an “NFT Market”. What kind of NFTs are you going to support (e.g. [Uniques module](https://github.com/paritytech/substrate/tree/master/frame/uniques), evm based standards, etc.)?

JhonnyAllen79 commented:
Thanks @Noc2, regarding that, we will archive it, we need a bridge to connect these two Blockchains in DEX, for NFT Market, the type of Nft we apply is drawing art, or standard based on Evm.

I will correct the proposal about the initial statement from you.

JhonnyAllen79 commented:
Hi Noc2, I've fixed and filled everything in the PR.  I hope the W3F team can review Segara's bikaproject.md, so that the Bika project can continue development.

Noc2 reviewed (changes requested):
Thanks for the update. However, your application still isn’t in the application folder. Also what kind of bridge are you going to use? Is there already an existing solution? Otherwise this seems to be the first thing, which you would need to develop. Are you going to use ink! or substrate for the dex or do you plan to develop this in solidity? Do you already have any mock-ups for the nft marketplace? In general, it seems to be a lot of work and looks more like two different grant applications. Also something to be aware of level 1 grants are only up to 10k.

JhonnyAllen79 commented:
Hi Noc2 thanks for the reply.
- Harmony and parachain, the bridge that will be used to connect the two Blockchains in the application. 
- We already have a good idea and planning to build the application.
- Yes we will develop solidity and use substrate for DEX development. 
- We will estimate it  about NFT we will also create an Nft market for the sale of NFT assets. 

Oh yeah sorry i don't know the limit of Grant level 1 limit i guess i need 10k for early development that makes sense.

JhonnyAllen79 commented:
Hi Noc2, I have fixed the requested grant amount as per level 1. and moved the application to the application folder.  I want BikaProject.md to be reviewed and get grants as per level 1.

Noc2 reviewed (changes requested):
Thanks for the reply. But if you want to exchange “Polkadot and BSC tokens”, you need an ethereum bridge between bsc and polkdot. Are you planning to use https://github.com/harmony-one/ethhmy-bridge? I’m a little bit confused on what you mean with Harmony. Also how is your substrate pallet dex different from any other dex pallets that already exist?

Noc2 reviewed (changes requested):
Btw. your file still isn’t in the application folder.

JhonnyAllen79 commented:
Hi Noc2, thanks for the reply, we will make a Dex with a new model with a different look from other Dex, namely novel features, etc. Yes we need an Ethereum bridge to connect polka dots and BSC, regarding Harmony we connect between several Blockchains to secure assets and for the benefit of web3, Bika wants to cooperate with Harmony and build it.

JhonnyAllen79 commented:
How do I put an application into a folder?

JhonnyAllen79 commented:
Sir, when will the Bika Bisa project get a grant from W3F, we will carry out project development this month, so we need funds for development.

Noc2 reviewed (changes requested):
You get the grant as soon as you get the required approvals on this PR. However keep in mind that we only pay you after the successful delivery of the milestone.  Also at the moment, the application is still missing some information or the deliverables are unclear to me: For example, what exactly is unique about your dex module compared to the uniswap implementation [here](https://github.com/lsaether/pallet-swaps) or any other existing dex pallet. What is the “Running test stage module”? Is this a substrate module? Etc.

JhonnyAllen79 commented:
Hi Noc, thanks for the answer, Bika project develops Dex with better model than Uniswap.  With the addition of features with the substrate module, running the module in the testing phase to find experiences and shortcomings that are lacking and need to be improved, I hope that W3F can provide a grant to Project Bika to create the first milestone. The Bika project requires level 1 funding because it is an Instant Grant, I hope W3F can help the Bika project.

JhonnyAllen79 commented:
I have put the application in the application folder as it should be, hopefully it can help speed up the review process so that the Bika project can get grant funds.

JhonnyAllen79 commented:
I hope W3F can provide a level 1 grant (instant grant), we need funds for project development and the launch of the first milestone, I hope you understand. I hope you can give the best for our small Project to grow and work.

Noc2 reviewed (changes requested):
Thanks for putting the file into the correct folder. I’m going to mark the application as ready for review and share it with the rest of the team. However, I personally won’t approve it, since it contains in my opinion very little concrete details as part of the specification (e.g. DEX), as I pointed out before. Nevertheless, I wish you all the best with your project.

JhonnyAllen79 commented:
Hi Noc2, thank you for changing the label, but we have explained about the direction and details about the project and so is the DEX, Bika will create a DEX with additional features namely novel, Nft Market, IDO, voting, swap/liquidity,  portfolio, and farm /staking, With the substrate module, we also use a bridge to load two Blockchains into the application. We hope that the Bika project can start and launch. The first milestone, I hope W3F can consider and provide grant assistance to the Bika project.

takahser reviewed (changes requested):
@JhonnyAllen79 Thanks for your application and your interest in the Web3 Foundation Grants Program.
After reading your application and the conversation above there are several questions that came up:

1. Your [webpage](https://bikatoken.com/) mentions the Bika Token, a BSC token, and its utilities and the project roadmap. However, Polkadot/Substrate/Parachains etc. are mentioned nowhere. Why is that?
2. In your application you write

    > The Bika project is a unique project and has a good ecosystem

   Could you perhaps give some examples of your current ecosystem (e.g. code libraries, products that use or intend to use your token, NFT artists/writers you partner with, etc.)?

3. You mention on several occasions that your product is "novel" and has "innovations". Could you give some concrete examples here as well?
4. From looking at your [diagram of the core components](https://drive.google.com/file/d/1ybTJF5hDS_zgS6HgXbGSR0WkTxV83oQW/view) it's not clear to me, how you're planning to exchange BSC and Polkadot tokens. Especially since the BSC blockchain can't even be found on the diagram. Could you elaborate on that and/or update the diagram?
5. In your application you write

    > Bika Token (BKT) is a Binance Smart chain(BSC) and Polkadot (DOT) based token

    Could you please explain which blockchain or ecosystem the token will be native to? Btw, Polkadot technically doesn't allow you to create tokens directly on their blockchain. Please explain or lmk if I'm missing something.

6. The [GitHub profiles](https://github.com/w3f/Grants-Program/pull/813/files#diff-2ea0600b8fca5ff1ebded3f2ebb9461597ad6651b309bddc2ce45ca4fd780928R138-R141) listed in your application have no repos containing any code. Can you provide us with some code your team wrote in the past so we can assess their experience?
7. Your legal address is Singapore, however, both LinkedIn profiles you linked are based in Indonesia. Is that correct or is it a mistake?

In my opinion the application currently doesn't explain the architecture of your DEX, how you want to integrate Polkadot with BSC, what you will use Substrate for, if you're planning to use ink! smart contracts, etc. which is why I'm currently also not willing to support it. Nevertheless I wish you all the best and success with your project!

JhonnyAllen79 commented:
Hi W3F, how about the grant proposal for the Bika  Project?  We respect all decisions from the W3F team, and we need grant funding for development as soon as possible.

Noc2 reviewed (changes requested):
Thanks for chasing us. So far it looks like the committee isn’t interested in supporting this project. But I will ping them again. As I pointed out earlier, I think you need to focus on specific aspects of the project and provide concrete technical details, why your dex or NFT solution is unique and why we should fund it.

JhonnyAllen79 commented:
Hi Noc2 thanks for the answer, we made a unique DEX and have new features like Nft Market, I think after filling it very clearly and this is a new innovation, we will create and run all Polkadot ecosystem for project development, we will develop community  and build ecosystem development for Polkadot and its substrates, Polkadot will be a partner and alliance to reach out to the development sector to achieve mutual benefit, Bika project is only asking for level 1 grant funding and we are just a small company so I hope W3F can help us in this. We will continue to reach out to W3F to cooperate and provide assistance for the Bika project, we will ensure a quality product.

JhonnyAllen79 commented:
Hi W3F, Come on fair W3F, I hope you can help us. We only ask for level 1 or lowest level grants for large project development, I hope W3F can discuss our request for this grant.

JhonnyAllen79 commented:
We have been waiting for this grant for a long time so we can work and do development, I hope W3F will appreciate it.

laboon commented:
I think the idea of "novels on the blockchain" has some merit, although this has been explored in a few other places already ( e.g. https://mirror.xyz/). I have reviewed this, but won't currently support it for the following reasons.

Going over the requirements ( https://github.com/w3f/Grants-Program )….

> It presents a well-researched or tested concept, for which ideally you are able to show some prior work.

The technical details are rather lacking. Ideally, I'd like to see a general software design description or specific list of requirements. I recommend reviewing some of the previously-accepted grants for a good idea of what is expected.

> Your team has proven experience with the relevant languages and technologies and/or a strong technical background. You will be asked to provide the GitHub profiles of your team members as part of your application, which we will examine these for past activity and code quality. Naturally, you can also link to projects on other platforms.

None of the Github profiles linked have significant usage (none of them have more than 8 commits) and https://github.com/arifark/Bika-Token has nothing but a placeholder README. Does the team have other experience that isn't shown in the Github profiles?

> Your application is rich in technical details and well-defined.

The technical detail is very high-level and not defined well. The architecture diagram linked does not explain how the different bridges will work.

> You can clearly present how your project stands out among competitors or implements technology that doesn't exist in the ecosystem yet.

I'm unclear how this stands out from using existing NFT marketplaces which support arbitrary data and are currently running on Kusama, or why the token/bridge with BSC is necessary, or how this code existing in the ecosystem would be useful to others.

That said, I'd be happy to look at it again if more details are provided, and wish you good luck on the project!

JhonnyAllen79 commented:
Hi W3F, we are tired of continuing this Proposal, we have been fighting for months or rather 3 months To be able to get a grant from W3F but we didn't get it, we are tired of being asked to provide details while we only ask for small funds and you ask for more for details  this high is not worth the funds we asked for, if W3F doesn't want to help the Bika project we don't mind, and we will look for another Blockchain company to work with to provide innovation in its Ecosystem. If there is no answer regarding this, we better back off and close this PR, if W3F has doubts and doesn't want to help. We have good details but we can't give them technically, they cost more than we ask. We will respect all decisions from W3F, if we don't get funding we will move to find a Blockchain company that can work with, thanks for your feedback.

JhonnyAllen79 commented:
We ask for small funds for the development of this large project is not an easy thing with this small fund we build a large project, we ask for small funds with the excuse that it is faster for us to receive the funds but we were wrong in thinking for instant level 1 funds. We can provide details and everything that W3F needs for the development of this project but we need funds for the development of this large project, but I see that the time we spent on this grant proposal has been very long and has no progress in getting funding from W3F, close the PR immediately if W3F doesn't want to help, we're tired of continuing if we don't get any help from W3F.

JhonnyAllen79 commented:
Hi Noc2, What do you think whether we can get a grant from W3F or not? 
We have been waiting to get this grant for a long time and to no avail.

Noc2 reviewed (changes requested):
As I said before, in its current form I’m not willing to support the project and it seems other members of the grants committee have similar concerns.  If you don’t want to provide more technical details, I suggest that we close the application. Also something to be aware of, we only pay you, once you actually deliver the milestones and not upfront.

JhonnyAllen79 commented:
Hi Noc2, We can provide technical details but here we need funds for project development, we only ask for small funds for large project development, we are willing to provide good details along with all the achievements but we need funds, I hope you understand, we are now This cannot provide it because we have not worked for 3 months because there is no clarity from W3F regarding our grant funds, and we have not received the funds, if W3F doesn't want to help, then we don't force it.

JhonnyAllen79 commented:
Please understand, we are just a small company that does not have funds we expect funds from W3F for us to work and do development.

JhonnyAllen79 commented:
I think you and the team have doubts about our project, we will not disappoint anyone, but I always insist that we are not a scam, we only ask for funds to work.

JhonnyAllen79 commented:
Why can't you provide upfront funds we need upfront funds for development? do you have any doubts about our project words? we have provided details in Pr. We are a small project just starting out and need financial assistance that's why we ask for funds upfront to do the work, we only ask for small funds for big project goals, I hope you can consider it and can help us. We beg for this. we only asked for 10k USD for a project of this size and you can't afford it.

Noc2 reviewed (changes requested):
I’m sorry, but we never pay teams upfront and only after the successful delivery of the milestone.

JhonnyAllen79 commented:
If after a successful milestone we will definitely ask for level 2 funds and above, sir (which is higher), that's why we try Level 1. Then we need a lot of time and funds to complete it.

JhonnyAllen79 commented:
Hi W3F, So how sir, what should i do? we need funds for the development of the Bika project.

semuelle commented:
> we need funds for the development of the Bika project.

There is a list of related grant programs in our README under alternative funding sources. I'm somewhat sceptical you will be satisfied with those either, because they usually don't pay upfront and will probably have similar questions/concerns about your application. 

Perhaps a crowd funding platform like Gitcoin might be better suited to your needs.

Noc2 reviewed (changes requested):
An alternative might also be to contact VCs and try to convince them about your idea. VCs usually require less technical details and provide upfront money. In any case, I wish you all the best with your project. I’m closing the application for now, since providing concrete technical details as well as being paid after the successful delivery are key requirements of our program. Maybe you can apply again at a later stage. Also let me know if I should reopen the application.

JhonnyAllen79 commented:
Hi Noc2, Our news how can we contact the VC?  we will try to contact and convince them that the project is utility and unique, try to explain how we can visit VC.

JhonnyAllen79 commented:
We will let you know when the Bika project has a milestone and concrete details.

JhonnyAllen79 commented:
Hi Noc2 do you have any suggestions on which VC I can contact?  we will convince them.
